src/main/java/org/jenkins/plugins/lockableresources/LockableResource.java

```diff
@@ -168,7 +184,25 @@ public String getReservedBy() {
 	}
 
 	@Exported
-	public boolean isReserved() {
+	public boolean isReserved() {	
```

Comment:
In LockableResource.isReserved, wouldn't it be more efficient to look at the constituents of the lab to see if one of them is a reservation? Since this seems like a common operation you might want to consider adding an operational state property to the lab objective that indicates that a reservation is present or not which you could interrogate directly.